### PR TITLE
Webpage layout

### PR DESCRIPTION
**Pull request by puteriffah**

### CODE REVIEW

**Comment by puteriffah** [comment]
ready

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-ruby-2.7.1
+ruby-2.7.1
```
Comment (saobv-2223) [inline .ruby-version]:
```suggestion
ruby-2.7.1

```

File: app/assets/stylesheets/colour.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+$light-gray: #777;
+$gray-medium-light: #eaeaea;
+$white: #fff;
+$greyish: #bbb;
+$red: red;
```
Comment (saobv-2223) [inline app/assets/stylesheets/colour.scss]:
```suggestion
$red: red;

```

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Course Registration"
+    page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```
Comment (saobv-2223) [inline app/helpers/application_helper.rb]:
```suggestion
  def full_title page_title
    base_title = "Course Registration"
    page_title.blank? ? base_title : "#{page_title} | #{base_title}"
```

File: app/helpers/static_pages_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (saobv-2223) [inline app/helpers/static_pages_helper.rb]: remove file that never used

**Review by saobv-2223 — COMMENTED** [review]
Your PR is having 2 commits, merge to 1 commit

**Comment by puteriffah** [comment]
ready

**Comment by puteriffah** [comment]
i have already fix my conflict, but there are still conflict occur

File: .gitignore [hunk 5]
```diff
@@ -33,6 +33,45 @@
 # Ignore master key for decrypting credentials and more.
 /config/master.key
 
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (saobv-2223) [inline .gitignore]: resolve conflict, please

File: Gemfile.lock [hunk 6]
```diff
@@ -58,6 +58,20 @@ GEM
       zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
+<<<<<<< HEAD
+=======
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (saobv-2223) [inline Gemfile.lock]: resolve conflict, please

File: app/views/layouts/application.html.erb [hunk 7]
```diff
@@ -1,15 +1,16 @@
...
   <head>
-    <title>MalayGroup3</title>
+    <title><%= full_title yield :title %></title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
-    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (saobv-2223) [inline app/views/layouts/application.html.erb]: usig "

**Comment by puteriffah** [comment]
ready

**Pull request merged**